src/ImpactX.cpp

```diff
@@ -97,6 +97,19 @@ namespace impactx
         amrex::ignore_unused(lev);
     }
 
+    /** Resize the mesh, based on the extent of the bunch of particle
+    */
+    void ImpactX::ResizeMesh () {
+        // Extract the mean and RMS size of the particle positions
+        amrex::ParticleReal x_mean, x_std, y_mean, y_std, z_mean, z_std;
+        mypc->MeanAndStdPositions(x_mean, x_std, y_mean, y_std, z_mean, z_std);
+        // Resize the domain size
+        amrex::RealBox rb(
+            {x_mean-3*x_std, y_mean-3*y_std, z_mean-3*z_std}, // Low bound
+            {x_mean+3*x_std, y_mean+3*y_std, z_mean+3*z_std}); // High bound
```

Comment:
As discussed with @qianglbl, this will actually be modified so as to use the min/max of the particle positions.